meta/views.py

```diff
@@ -2071,14 +2072,18 @@ def authors_page(request):
     '''Page showing the full list of authors and specialists.'''
 
     # Get all person instances associated to media as authors
-    authors = Person.objects.filter(media_as_author__isnull=False).distinct()
+    authors = Person.objects.filter(id__in=Media.objects.values('authors')).order_by('name')
```

Comment:
The standard for this type of query is checking if the `related_name` is null:

```
authors = Person.objects.filter(media_as_author__isnull=False).distinct()
specialists = Person.objects.filter(media_as_specialist__isnull=False).distinct()
curators = Person.objects.filter(media_as_curator__isnull=False).distinct()
```

And the `order_by` is in principle not needed because the default ordering of `Person` is by name.

Reply:
OK! Changed.